specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/workbooks_API.json

```diff
@@ -0,0 +1,808 @@
+          "type": "string",
+          "description": "The kind of workbook. Choices are user and shared.",
+          "enum": [
+            "user",
+            "shared"
+          ],
+          "x-ms-enum": {
+            "name": "kind",
+            "modelAsString": true
+          }
+        },
+        "etag": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "type": "string",
+          "description": "Resource etag"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "description": "An azure resource object"
+    },
+    "WorkbooksListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Workbook"
+          },
+          "description": "An array of workbooks."
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "type": "array",
+      "description": "Workbook list result."
+    },
+    "Workbook": {
+      "description": "An Application Insights workbook definition.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/WorkbookResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a workbook for an Azure resource.",
+          "$ref": "#/definitions/WorkbookProperties"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        }
+      }
+    },
+    "WorkbookProperties": {
+      "description": "Properties that contain a workbook.",
+      "type": "object",
+      "required": [
+        "displayName",
+        "category",
+        "serializedData"
+      ],
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The user-defined name (display name) of the workbook."
+        },
+        "serializedData": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "Configuration of this particular workbook. Configuration data is a string containing valid JSON"
+        },
+        "version": {
+          "type": "string",
+          "description": "Workbook version"
+        },
+        "timeModified": {
+          "type": "string",
+          "description": "Date and time in UTC of the last modification that was made to this workbook definition.",
+          "readOnly": true
+        },
+        "category": {
+          "type": "string",
+          "description": "Workbook category, as defined by the user at creation time."
+        },
+        "tags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of 0 or more tags that are associated with this workbook definition"
+        },
```

Comment:
can we either remove the "tags" in the "properties" field or make it optional or obsolete?